documentation/modules/proc-installing-using-template.adoc

```diff
@@ -5,7 +5,7 @@
 [id='installing-using-template-{context}']
 = Installing {ProductName} using OpenShift Template
 
-Installing {ProductName} using the OpenShift Template is useful for evaluating {ProductName}. For a production setup, it is recommended to follow xref:installing-using-ansible-{context}[] or xref:installing-using-manual-steps-{context}[].
+Installing {ProductName} using the OpenShift Template is useful for evaluating {ProductName}. For a production setup, it is recommended to follow xref:installing-using-bundle-{context}[], xref:installing-using-ansible-{context}[], or xref:installing-using-manual-steps-{context}[].
```

Comment:
Lowercase "Template"

For a production deployment, it is recommended to use one of the following installation methods instead:
* xref:installing-using-bundle-{context}[]
* xref:installing-using-ansible-{context}[]
* xref:installing-using-manual-steps-{context}[]